src/sdk/services/blockchain/contracts/IDContract.ts

```diff
@@ -6,6 +6,8 @@ import { ActionData, Signer, transact } from '../eosio/transaction';
 import { SdkErrors, throwError } from '../../../util/errors';
 import { sha256 } from '../../../util/crypto';
 
+const contractName = 'id.tmy';
```

Comment:
UPPER_SNAKE_CASE for a global constant like this
https://github.com/Tonomy-Foundation/Tonomy-ID-Integration/blob/feature/213-naming-convention/JAVASCRIPT_CONVENTIONS.md#general